### PR TITLE
[hail] (PTypes) Convert `physicalType` usages in tests

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/src/test/scala/is/hail/annotations/UnsafeSuite.scala [hunk 1]
```diff
@@ -309,19 +311,19 @@ class UnsafeSuite extends HailSuite {
       tv.typeCheck(a2)
 
       region.clear()
-      rvb.start(tv.physicalType.fundamentalType)
+      rvb.start(t)
```
Comment (akotlar) [inline hail/src/test/scala/is/hail/annotations/UnsafeSuite.scala]: Could you help me understand why this is no longer calling fundamentalType (i.e t.fundamentalType). Same for rvb2.start

Comment (tpoterba) [reply hail/src/test/scala/is/hail/annotations/UnsafeSuite.scala]: unnecessary:
```
  def start(newRoot: PType) {
    assert(inactive)
    root = newRoot.fundamentalType
  }
```

Comment (akotlar) [reply hail/src/test/scala/is/hail/annotations/UnsafeSuite.scala]: Thanks!

**Comment by akotlar** [comment]
You have a few failures, rvb.addAnnotation(t, a) needs to be rvb.addAnnotation(t.virtualType, a)

**Comment by akotlar** [comment]
Updates needed for the following:

```sh
> Task :compileTestScala
Pruning sources from previous analysis, due to incompatible CompileSetup.
/io/repo/hail/src/test/scala/is/hail/annotations/ScalaToRegionValue.scala:9: type mismatch;
 found   : is.hail.expr.types.physical.PType
 required: is.hail.expr.types.virtual.Type
    rvb.addAnnotation(t, a)
                      ^
/io/repo/hail/src/test/scala/is/hail/annotations/StagedRegionValueSuite.scala:487: type mismatch;
 found   : is.hail.expr.types.physical.PType
 required: is.hail.expr.types.physical.PBaseStruct
        SafeRow(t, region, copyOff)
                ^
/io/repo/hail/src/test/scala/is/hail/expr/ir/Aggregators2Suite.scala:46: type mismatch;
 found   : is.hail.expr.types.virtual.TStruct
 required: is.hail.expr.types.physical.PType
      val argOff = ScalaToRegionValue(region, argT, argVs)
                                              ^
/io/repo/hail/src/test/scala/is/hail/expr/ir/Aggregators2Suite.scala:177: type mismatch;
 found   : is.hail.expr.types.virtual.Type
 required: is.hail.expr.types.physical.PType
        val voff = ScalaToRegionValue(region, stream.typ, lit)
                                                     ^
/io/repo/hail/src/test/scala/is/hail/nativecode/RegionValueIteratorSuite.scala:73: type mismatch;
 found   : is.hail.expr.types.physical.PTuple
 required: is.hail.expr.types.virtual.Type
          rvb.addAnnotation(t, r)
                            ^
/io/repo/hail/src/test/scala/is/hail/nativecode/RegionValueIteratorSuite.scala:133: type mismatch;
 found   : is.hail.expr.types.physical.PTuple
 required: is.hail.expr.types.virtual.Type
Error occurred in an application involving default arguments.
          rvb.addAnnotation(t, r)
```

**Comment by tpoterba** [comment]
thanks, will fix

**Pull request merged**